build/calico-cni-plugin/calico-cni-plugin.spec.tmpl

```diff
@@ -0,0 +1,53 @@
+%global provider        github
+%global provider_tld    com
+%global project         projectcalico
+%global repo            cni-plugin
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+
+%ifarch x86_64
+%global built_arch              amd64
+%global calico_sha256           5736d46dc8f3d17eafd1d7874c49457244b698d5fccf726066c60c136ac5a318
```

Comment:
So, `CALICO_VERSION` gets templated (at only one location), but these aren't. I.e. we can't change `CALICO_VERSION`, things would break anyway.